os/booting-on-interoute.md

```diff
@@ -1,196 +1,266 @@
+To be able to make an SSH connection from the Internet to the virtual machine a port forwarding rule for the network must be created.
 
 ```sh
 > createPortForwardingRule \
     protocol=TCP \
     publicport=22 \
-    ipaddressid=value1 \
-    virtualmachineid=value2 \
+    ipaddressid=b2b68408-76a9-4dc7-9a2f-f3cf10616aca \
+    virtualmachineid=e08a3199-9d16-4244-aa89-23395d9627d7 \
     privateport=22 \
     openfirewall=true
 ```
-The last configuration step is to set an egress firewall rule for the network so that the CoreOS VM will be able to get outward access to the internet. This is needed for Docker to access repositories for container images, and to allow CoreOS to access internet update servers to do automatic updating:
 
+The 'virtualmachineid' can be read from the deploy output above. 'ipaddressid' can be found with the API command listPublicIpAddresses, and (if there is more than one public IP address) you look for the 'associatednetworkid' matching the id of the network that the virtual machine is attached to:
 
 ```sh
-> createEgressFirewallRule networkid=e9e1220b-76c8-47cd-a6c2-885ffee49972 \
-    protocol=all \
-    cidr=0.0.0.0/0
+> listPublicIpAddresses zoneid=a5d3e015-0797-4283-b562-84feea6f66af filter=id,ipaddress,associatednetworkid
+count = 1
+publicipaddress:
++-----------------+--------------------------------------+--------------------------------------+
+|   ipaddress     |         associatednetworkid          |                  id                  |
++-----------------+--------------------------------------+--------------------------------------+
+| 213.XXX.XXX.185 | c5841e7c-e69e-432b-878b-c108b07a160f | b2b68408-76a9-4dc7-9a2f-f3cf10616aca |
++-----------------+--------------------------------------+--------------------------------------+
 ```
 
-Note that allowing traffic from any IP is not a good practice.
+(The full IP address has been Xed out for privacy reasons.)
+
+The last configuration step is to create an egress firewall rule for the network so that the CoreOS virtual machine will be able to get outward access to the Internet. This is needed to allow CoreOS to access update servers to do automatic updating, and for Docker to access repositories for container images:
+
+```sh
+> createEgressFirewallRule networkid=c5841e7c-e69e-432b-878b-c108b07a160f protocol=all cidr=0.0.0.0/0
+```
 
-So finally the VM is set up for connection, using the 'ipaddress' found from the listPublicIpAddresses command and specifying the private SSH key file to match the public key which I registered in VDC. Type this ssh command into a terminal:
+(Note that these network rules are simple and permissive, while rules for production virtual machines should always be more strictly defined.)
 
+The virtual machine is now set up for connection, using the 'ipaddress' found above and specifying the private SSH key file to match the public key which was registered in VDC (and there is no root user in CoreOS, the default user is 'core'):
```

Comment:
Change to:
"The virtual machine is now set up for connection, using the `ipaddress` found above, and specifying the private SSH key file to match the public key which was registered in VDC. Note that there is no `root` user in CoreOS; the default user is named `core`:"